specification/network/resource-manager/Microsoft.Network/2017-06-01/expressRouteConnector.json

```diff
@@ -0,0 +1,412 @@
+			}
+		},
+	 "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/expressRouteConnectors": {
+		  "get": {
+			"tags": [
+			  "ExpressRouteConnectors"
+			],
+			"operationId": "ExpressRouteCircuitConnectors_ListByResourceGroup",
+			"x-ms-examples": {
+                    "RouteFilterListByResourceGroup": { "$ref": "./examples/ExpressRouteConnectorListByResourceGroup.json" }
+            },
+			"description": "Gets all express route Connector resources in a given resource group and subscription.",
+			"parameters": [
+			  {
+				"$ref": "#/parameters/ResourceGroupNameParameter"
+			  },
+			  {
+				"$ref": "#/parameters/ApiVersionParameter"
+			  },
+			  {
+				"$ref": "#/parameters/SubscriptionIdParameter"
+			  }
+			],
+			"responses": {
+			  "200": {
+				"description": "Request successful. The operation returns a list of Express Route Connector resources.",
+				"schema": {
+				  "$ref": "#/definitions/expressRouteConnectorListResult"
+				}
+			  }
+			},
+			"x-ms-pageable": {
+			  "nextLinkName": "nextLink"
+			}
+		  }
+		},	
+	"/subscriptions/{subscriptionId}/providers/Microsoft.Network/expressRouteConnectors": {
+		  "get": {
+			"tags": [
+			  "ExpressRouteConnectors"
+			],
+			"operationId": "ExpressRouteCircuitConnectors_List",
+			"description": "Gets all express route Connector resources in a given subscrption.",
+			"parameters": [
+			  {
+				"$ref": "#/parameters/ApiVersionParameter"
+			  },
+			  {
+				"$ref": "#/parameters/SubscriptionIdParameter"
+			  }
+			],
+			"responses": {
+			  "200": {
+				"description": "Request successful. The operation returns a list of Express Route Connector resources.",
+				"schema": {
+				  "$ref": "#/definitions/expressRouteConnectorListResult"
+				}
+			  }
+			},
+			"x-ms-pageable": {
+			  "nextLinkName": "nextLink"
+			}
+		  }
+		}	
+	},	 
+	"definitions": {	
+		"expressRouteConnectorPropertiesFormat": {
+			"required": [
+				"circuits",
+				"addressPrefix"
+			],
+			"properties": {
+				"circuits": {
+					"type": "array",
+					"items": {
+						"$ref": "./network.json#/definitions/SubResource",
+						"description": "The reference to the express route circuit resource."
+					},
+					"description": "List of Express Route Circuits."
+				},
+				"authorizations": {
+					"type": "array",
+					"items": {
+						"type": "string",
+						"description": "Authorization Key"
+					},
+					"description": "List of Circuit Authorization Keys."
+				},
+				"addressPrefix": {
+				  "type": "string",
+				  "description": "The reference of the addressPrefix resource."
+				},
+				"connectionStatus": {
+					"type": "string",
+					"readOnly": true,
+					"description": "Specifies the state of Express Route Connector Link : Succeeded / Failed / In Progress."
+				}
+			},	
+			"description": "Properties of Express Route Connector Resource."
+		},
+		"expressRouteConnector": {
+		  "properties": {
+			"location": {
```

Comment:
Properties `location`, `tags` and `name` are inherited by `Resource`! Don't specify those here.